src/main/java/bio/terra/app/controller/converters/SqlSortDirectionConverterDesc.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.app.controller.converters;
+
+import bio.terra.model.SqlSortDirectionDescDefault;
+import java.util.Arrays;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SqlSortDirectionConverterDesc
```

Comment:
Should this instead put `Converter` at the end?
```suggestion
public class SqlSortDirectionDescDefaultConverter
```